samples/runtime/qpic-simulator/simulator/Simulator.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
You may be able to combine several of these files together since the QpicSimulator is shorter than some of the other simulators that motivate using `partial class` in other contexts.

Reply:
I did this initially, but found this easier to get a good overview of the classes involved. Also, I might add one or two more internal classes later on, e.g., to support mixed-polarity multiple-controlled gates.